Detector/DetFCCeeCLD/compact/FCCee_o1_v05/InnerTracker.xml

```diff
@@ -0,0 +1,613 @@
+                <ring r="InnerTracker_Endcap_radius_3+22*ITM" zstart="InnerTracker_Endcap_z_3" nmodules="24" dz="3*mm" module="InnerTrackerEndcapModule_12x8_In" phi0="0"/>
+            </layer>
+
+
+            <layer id="4">
+                <ring r="InnerTracker_Endcap_radius_4"        zstart="InnerTracker_Endcap_z_4" nmodules="24" dz="3*mm" module="InnerTrackerEndcapModule_6x4_Out"  phi0="0"/>
+                <ring r="InnerTracker_Endcap_radius_4+4*ITM"  zstart="InnerTracker_Endcap_z_4" nmodules="24" dz="3*mm" module="InnerTrackerEndcapModule_8x8_Out"  phi0="0"/>
+                <ring r="InnerTracker_Endcap_radius_4+12*ITM" zstart="InnerTracker_Endcap_z_4" nmodules="24" dz="3*mm" module="InnerTrackerEndcapModule_9x5_Out"  phi0="0"/>
+                <ring r="InnerTracker_Endcap_radius_4+17*ITM" zstart="InnerTracker_Endcap_z_4" nmodules="24" dz="3*mm" module="InnerTrackerEndcapModule_10x4_Out"  phi0="0"/>
+                <ring r="InnerTracker_Endcap_radius_4+21*ITM" zstart="InnerTracker_Endcap_z_4" nmodules="24" dz="3*mm" module="InnerTrackerEndcapModule_11x3_Out" phi0="0"/>
+                <ring r="InnerTracker_Endcap_radius_4+24*ITM" zstart="InnerTracker_Endcap_z_4" nmodules="24" dz="3*mm" module="InnerTrackerEndcapModule_12x3_Out" phi0="0"/>
+            </layer>
+
+
+            <layer id="5">
+                <ring r="InnerTracker_Endcap_radius_5"        zstart="InnerTracker_Endcap_z_5" nmodules="24" dz="3*mm" module="InnerTrackerEndcapModule_7x5_In"  phi0="0"/>
+                <ring r="InnerTracker_Endcap_radius_5+5*ITM"  zstart="InnerTracker_Endcap_z_5" nmodules="24" dz="3*mm" module="InnerTrackerEndcapModule_9x8_In"  phi0="0"/>
+                <ring r="InnerTracker_Endcap_radius_5+13*ITM" zstart="InnerTracker_Endcap_z_5" nmodules="24" dz="3*mm" module="InnerTrackerEndcapModule_10x4_In"  phi0="0"/>
+                <ring r="InnerTracker_Endcap_radius_5+17*ITM" zstart="InnerTracker_Endcap_z_5" nmodules="24" dz="3*mm" module="InnerTrackerEndcapModule_11x6_In"  phi0="0"/>
+            </layer>
+
+            <layer id="6">
+                <ring r="InnerTracker_Endcap_radius_6"        zstart="InnerTracker_Endcap_z_6" nmodules="24" dz="3*mm" module="InnerTrackerEndcapModule_7x5_Out"  phi0="0"/>
+                <ring r="InnerTracker_Endcap_radius_6+5*ITM"  zstart="InnerTracker_Endcap_z_6" nmodules="24" dz="3*mm" module="InnerTrackerEndcapModule_9x3_Out"  phi0="0"/>
+                <ring r="InnerTracker_Endcap_radius_6+8*ITM"  zstart="InnerTracker_Endcap_z_6" nmodules="24" dz="3*mm" module="InnerTrackerEndcapModule_9x5_Out"  phi0="0"/>
+                <ring r="InnerTracker_Endcap_radius_6+13*ITM" zstart="InnerTracker_Endcap_z_6" nmodules="24" dz="3*mm" module="InnerTrackerEndcapModule_12x8_Out" phi0="0"/>
+            </layer>
+
+        </detector>
+
+<!--
+	<detector name="InnerTrackerBarrelSupport" type="TrackerBarrelSupport_o1_v01" id="0"  reflect="true" region="InnerTrackerBarrelRegion">
```

Comment:
Should be 
```suggestion
	<detector name="InnerTrackerBarrelSupport" type="TrackerBarrelSupport_o1_v02" id="0"  reflect="true" region="InnerTrackerBarrelRegion">
```
see https://github.com/iLCSoft/lcgeo/pull/259

Reply:
Hi Valentin,

modifying this to v02 i get an error when building the detector: "No factory with name Create(TrackerBarrelSupport_o1_v02) for type TrackerBarrelSupport_o1_v02 found." 

what am I missing?